core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -1881,11 +1911,11 @@ public void testCreateTableAsSelect()
                         new FunctionCall(QualifiedName.of("concat"), ImmutableList.of(new StringLiteral("ban"), new StringLiteral("ana")))),
                 new Property(new Identifier("a"), new Array(ImmutableList.of(new StringLiteral("v1"), new StringLiteral("v2")))));
         assertThat(statement("CREATE TABLE foo(x,y) COMMENT 'test'" +
-                "WITH ( \"string\" = 'bar', \"long\" = 42, computed = 'ban' || 'ana', a = ARRAY[ 'v1', 'v2' ] ) " +
-                "AS " +
-                "SELECT a,b FROM t " +
-                "WITH NO DATA")).ignoringLocation().isEqualTo(
-                        new CreateTableAsSelect(table, querySelectColumns, false, delimitedProperties, false, Optional.of(ImmutableList.of(new Identifier("x"), new Identifier("y"))), Optional.of("test")));
+                        "WITH ( \"string\" = 'bar', \"long\" = 42, computed = 'ban' || 'ana', a = ARRAY[ 'v1', 'v2' ] ) " +
+                        "AS " +
+                        "SELECT a,b FROM t " +
+                        "WITH NO DATA")).ignoringLocation().isEqualTo(
+                                new CreateTableAsSelect(table, querySelectColumns, false, false, delimitedProperties, false, Optional.of(ImmutableList.of(new Identifier("x"), new Identifier("y"))), Optional.of("test")));
```

Comment:
The formatting changes should go to a separate commit.

Reply:
They are already in a separate commit `Refactor testCreateTableAsSelect` to use non-deprecated methods. Isn't that already OK?

Reply:
The indentation change seems to be part of the `Support CREATE OR REPLACE in parser` commit.